chargeOclock/Source/Other/NetworkAPI.swift

```diff
@@ -39,6 +48,14 @@ class NetworkAPI {
 	private init() {
 	}
 
+	fileprivate func performPostRequest(to uri: String, json: Data? = nil, completion: @escaping NetworkCompletion) {
```

Comment:
don't mix json and data... if you name something json, let it be a dictionary of type `[String: Any]`, if you use the `Data` type, name it data